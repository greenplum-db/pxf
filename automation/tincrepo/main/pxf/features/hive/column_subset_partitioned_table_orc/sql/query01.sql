-- @description query01 for PXF Hive Small Column subset test

SELECT * from pxf_hive_small_data WHERE num1 <= 10 ORDER BY num1;
