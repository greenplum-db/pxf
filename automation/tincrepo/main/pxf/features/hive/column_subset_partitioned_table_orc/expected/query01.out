-- @description query01 for PXF Hive Small Column subset test
SELECT * from pxf_hive_small_data WHERE num1 <= 10 ORDER BY num1;
  fmt  | num1 |  t2  
-------+------+------
 abcd  |    1 | s_6
 abcde |    2 | s_7
 abcde |    3 | s_8
 abcde |    4 | s_9
 abcde |    5 | s_10
 abcde |    6 | s_11
 abcde |    7 | s_12
 abcde |    8 | s_13
 abcde |    9 | s_14
 abcde |   10 | s_15
(10 rows)
